app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb

```diff
@@ -116,6 +116,8 @@ def get_load_balancer_pool_members
   end
 
   def get_load_balancer_listeners
+    # Initialize the data, otherwise the saving code will not catch this when empty
+    @data[:load_balancer_listeners] ||= []
```

Comment:
so it's likely that we might need to backport it.
can you put this in a PR of its own then, because it can be backported without the other stuff then